instrumentation/micrometer/micrometer-1.5/library/src/main/java/io/opentelemetry/instrumentation/micrometer/v1_5/PrometheusModeNamingConvention.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.micrometer.v1_5;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.config.NamingConvention;
+import javax.annotation.Nullable;
+
+enum PrometheusModeNamingConvention implements NamingConvention {
```

Comment:
AFAICT, the difference between this and the upstream naming convention is we don't append _total to counters, since the prometheus exporter does it. We should probably document that